src/index.js

```diff
@@ -8,6 +8,19 @@ class Animation extends PureComponent {
 
   componentDidMount() {
     this.loadAnimation(this.props);
+
+    if (this.props.progress) {
```

Comment:
Maybe also check if progress is an object that has an addListener?

And handle the case where progress is a simple number too?

Otherwise please indicate in the readme that the progress prop is partially implemented (must be an Animated.value, not a number)

Reply:
Fixed, thanks.